mainnet-1/specs/cosmoshub.json

```diff
@@ -0,0 +1,159 @@
+{
+  "proposal": {
+    "title": "Add Specs: Cosmos hub",
+    "description": "Adding new specification support for relaying cosmoshub data on Lava",
+    "specs": [
+      {
+        "index": "COSMOSHUB",
+        "name": "cosmos hub mainnet",
+        "enabled": true,
+        "imports": [
+          "COSMOSSDKFULL",
```

Comment:
cosmossdk spec full is for v47, cosmos hub is already v50, we need to go over the apis, check what is missing and add them